doc/installation.rst

```diff
@@ -28,11 +28,11 @@ download the compressed archive from the *develop* branch of
 Expand the archive.
 Inside is a script named ``setup.py``.
 Execute ``python setup.py install``
```

Comment:
Maybe change this to `pip install .` to avoid encouraging use of deprecated `setup.py`?

Reply:
See other comment on this file.